RunLog/RunLog/Sources/Presentation/Log/View/Timeline/TimelineViewCell.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  TimelineViewCell.swift
+//  RunLog
+//
+//  Created by 신승재 on 3/16/25.
+//
+
+import UIKit
+
+class TimelineViewCell: UITableViewCell {
+    
+    static let identifier = "TimelineCell"
+    
+    // MARK: - UI Component
+    private let cellContainer = UIView().then {
+        $0.backgroundColor = .Gray700
+        $0.layer.cornerRadius = 8
+    }
+    
+    lazy var distanceLabel = UILabel().then {
+        $0.attributedText = .RLAttributedString(
+            text: "25km",
+            font: .Heading4,
+            color: .LightPink
+        )
+    }
+    
+    lazy var titleLabel = UILabel().then {
+        $0.attributedText = .RLAttributedString(
+            text: "하트런",
+            font: .Heading1,
+            color: .Gray000
+        )
+    }
+    
+    lazy var dateLabel = UILabel().then {
+        $0.attributedText = .RLAttributedString(
+            text: "2025. 02. 03.",
+            font: .Label2,
+            color: .Gray200
+        )
+    }
+    
+    lazy var trackImageView = UIView().then {
+        // TODO: ImageView로 바꾸기
+        $0.backgroundColor = .black
+        $0.layer.cornerRadius = 4
+    }
+    
+    // MARK: - Init
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupUI()
+        setupLayout()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been impl")
+    }
+
+
+    // MARK: - Setup UI
+    private func setupUI() {
+        cellContainer.addSubviews(distanceLabel, titleLabel, dateLabel, trackImageView)
+        self.addSubviews(cellContainer)
+    }
+    
+    // MARK: - Setup Layout
+    private func setupLayout() {
+        cellContainer.snp.makeConstraints {
+            $0.top.leading.trailing.equalToSuperview()
+            $0.bottom.equalToSuperview().offset(-18)
+        }
+        
+        distanceLabel.snp.makeConstraints {
+            $0.height.equalTo(49)
+            $0.leading.equalToSuperview().offset(24)
+            $0.top.equalToSuperview().offset(24)
```

Comment:
```suggestion
            $0.top.leading.equalToSuperview().offset(24)
```
혹시 따로 지정해준 이유가 있으실까요?